managed/models/postgresql_version_test.go

```diff
@@ -78,7 +78,7 @@ func TestGetPostgreSQLVersion(t *testing.T) {
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
+			// t.Parallel()
```

Comment:
Problematic test?

Reply:
Yes, permanently racing, so this was the only way to make them pass :(